### DISCUSSION TITLE
Google Cloud Run ?

### DISCUSSION THREAD

**Discussion by liohe** [stackkit/laravel-google-cloud-tasks-queue#9] [General] [answered]
Hi,
Does it supports Google Cloud Run ?
Thanks,
Lio

**Comment by marickvantuil** [comment, marked as answer]
Hey, this package does indeed support Cloud Run, which is what I have used to develop it. In fact, this package should be platform agnostic. Doesn't matter if it's hosted on a VPS or in a container.
